modules/transactions/src/main/java/org/apache/ignite/internal/tx/views/TransactionsViewProvider.java

```diff
@@ -108,6 +109,7 @@ public Iterator<TxInfo> iterator() {
                     new TransformingIterator<>(roTxIds.iterator(), TxInfo::readOnly),
                     rwTxStates.entrySet().stream()
                             .filter(e -> localNodeId.equals(e.getValue().txCoordinatorId())
+                                    && e.getValue().tx() != null && !e.getValue().tx().isReadOnly()
```

Comment:
Can `e.getValue().tx() == null` and `ocalNodeId.equals(e.getValue().txCoordinatorId())` at the same time?

Reply:
Of course, we can skip ` localNodeId.equals(e.getValue().txCoordinatorId())` checking and also use `rwTxStates` for both transaction types. But I left the logic as is because it takes additional changes that are not related for this PR.